vtexdocs/devportal
# Feature/OpenAPI Rendering

#### What is the purpose of this pull request?

To add RapiDoc dependency and example pages of API specification files

#### What problem is this solving?

The Developers Portal should have pages for VTEX's API specification files

#### How should this be manually tested?

- Go to [this page](https://deploy-preview-13--xenodochial-shockley-a99d82.netlify.app/) and try clicking on the API Reference card of the Documentation section
- You should see a list containing two links, try clicking them both
- Each one should lead to a different page, one that renders the contents of its correspondent file (`/public/docs/catalog.json` for Catalog API and `/public/docs/checkout.json` for Checkout API)

#### Screenshots or example usage

https://user-images.githubusercontent.com/37647055/161334343-fe6634bc-9be5-4721-be68-9472eb7c4113.mp4

#### Types of changes
